src/index.js

```diff
@@ -48,7 +48,36 @@ const parsers = {
 const printers = {
   php: {
     print,
-    massageAstNode: clean
+    massageAstNode: clean,
+    getCommentChildNodes(node) {
+      let children = node.children || node.body || node.traits;
+      if (!children) {
+        return [];
+      }
+      if (children.kind && children.kind === "block") {
```

Comment:
I guess this would be equivalent:`if (children.kind === "block") {`

do you prefer the longer version?

Reply:
@nicoder It is PoC, but you are right :+1: